src/redmodding.ts

```diff
@@ -36,11 +37,13 @@ import {
   VortexApi,
   VortexDiscoveryResult,
   VortexExtensionContext,
+  VortexIToolShim,
   VortexState,
   VortexToolDiscovered,
 } from "./vortex-wrapper";
 
-// This function runs on starting up Vortex or switching to Cyberpunk as the active game. This may need to be converted to a test, but the UI for tests is less flexible.
+// This function runs on starting up Vortex or switching to Cyberpunk as the active game.
+// This may need to be converted to a test, but the UI for tests is less flexible.
```

Comment:
I feel like this comment is misplaced...

Reply:
Based on the original file: https://github.com/E1337Kat/cyberpunk2077_ext_redux/blob/d7c7dd93b1266f4cc8f628b5eecbccfccf85203a/src/prepareForModding.ts, I believe the comment referred to the default exported function prepareForModding, which is now the wrapped `wrappedPrepareForModdingWithREDmodding` or the "internal" `prepareForModdingWithREDmodding` I thing adding the comment as the comment doc to the wrapped function would be a good change to make it obvious what is going on here.

Reply:
(I know that isn't a part of this PR, but this change made me think of it)